packages/astro/test/astro-assets-prefix.test.js

```diff
@@ -72,7 +77,10 @@ describe('Assets Prefix - with path prefix', () => {
 	before(async () => {
 		fixture = await loadFixture({
 			root: './fixtures/astro-assets-prefix/',
+			outDir: './dist/server',
 			build: {
+				client: './dist/server/client',
+				server: './dist/server/server',
 				assetsPrefix: '/starting-slash',
```

Comment:
Since we haven't found a way to avoid changing `outer`, `build.client` and `build.server` manually, can you update the contribution guide and explains this? 

We haven't been doing an excellent job updating the contribution guide, but I can help with that. I think this quirk should be documented so developers know how to troubleshoot the issue in case new tests will be added.